src/db-service.ts

```diff
@@ -250,4 +276,26 @@ export class ItemValidationService {
       )
       .then(({ rows }) => rows[0]);
   }
+
+    /**
+   * Update the enabled field of an item validation process
+   * @param id process id
+   * @param value string TRUE or FALSE
+   */
+     async updateItemValidationProcess(
```

Comment:
You should change the naming for `toggleEnabledForItemValidationProcess` or similar.